bool app_enabled
string app_msg

string location_str

bool classifier_running

string image_topic

string[] available_classes_list
string[] selected_classes_list

float32 alert_delay_sec 
float32 clear_delay_sec

float32 trigger_delay_sec 
bool snapshot_trigger_enabled
bool event_trigger_enabled
